docs/OpenTelemetry.md

```diff
@@ -1,12 +1,13 @@
 # OpenTelemetry
 
-**Supported tracing frameworks**:
+**Supported OpenTelemetry features**:
 
 | Type          | Documentation                                        | datadog version | Gem version support |
 | ------------- | ---------------------------------------------------- | --------------- | ------------------- |
-| OpenTelemetry | https://github.com/open-telemetry/opentelemetry-ruby | 1.9.0+          | >= 1.1.0            |
+| Tracing       | https://github.com/open-telemetry/opentelemetry-ruby | 1.9.0+          | >= 1.1.0            |
+| Metrics       | https://github.com/open-telemetry/opentelemetry-ruby | 2.24.0+          | Metrics SDK >= 0.8, Exporter >= 0.4 |
```

Comment:
```suggestion
| Metrics       | https://github.com/open-telemetry/opentelemetry-ruby | 2.24.0+          | [Metrics SDK](https://rubygems.org/gems/opentelemetry-metrics-sdk) >= 0.8, [OTLP Metrics Exporter](https://rubygems.org/gems/opentelemetry-exporter-otlp-metrics) >= 0.4 |
```
I think we should make it clear which libraries with a panel, specially for the exporter, since there are multiple OTel gems with exporter in the name: https://rubygems.org/search?query=opentelemetry-exporter